### PR TITLE
Expose ORD Version Through Well-Known Endpoint

### PR DESCRIPTION
**Pull request by SvetlinZarev-SAP**
**Description**

Changes proposed in this pull request:
* Expose the implemented ORD version in the /info endpoint

**Related issue(s)**

- CMP-1679

**Pull Request status**

- [X] Implementation
- [X] Unit tests
- [X] Integration tests
- [X] `chart/compass/values.yaml` is updated 
- [ ] Mocks are regenerated, using the automated script

### CODE REVIEW

**Review by dzahariev — APPROVED** [review]

**Review by dzahariev — DISMISSED** [review]

**Comment by dzahariev** [comment]
Looking at history of change https://github.com/kyma-incubator/compass/pull/2262, I can see that job pre-main-compass-integration became unstable. Since then the job needs to be triggered multiple times in order to pass.

File: components/director/internal/info/handler.go [hunk 1]
```diff
@@ -7,21 +7,42 @@ import (
 	"github.com/kyma-incubator/compass/components/director/pkg/httputils"
 )
 
+// Implemented ORD Version
+const ordVersion = "1.1.0"
```
Comment (DimitarPetrov) [inline components/director/internal/info/handler.go:11]: It is probably better to move this constant to the open_resource_discovery package and export it.

On one hand, it is domain-specific and fits better, on the other hand, if you add it here on each version increasing a review from team-raptor will be needed.

**Review by DimitarPetrov — CHANGES_REQUESTED** [review]

File: components/director/internal/info/handler.go (hunk 1 above)
Comment (SvetlinZarev-SAP) [reply components/director/internal/info/handler.go]: done

**Review by DimitarPetrov — APPROVED** [review]

**Review by alextargov — APPROVED** [review]

**Review by DimitarPetrov — APPROVED** [review]

**Pull request merged**